docs/source/common/checkpointing_expert.rst

```diff
@@ -8,6 +8,13 @@ Checkpointing (expert)
 
 TODO: I don't understand this...
 
+*********************************
+Writing your own Checkpoint class
+*********************************
+
+We provide ``BaseModelCheckpoint`` class, for easier subclassing. Users may want to subclass it in case of writing custom ``ModelCheckpoint`` callback, so that the ``Trainer`` recognizes the custom class as a checkpointing callback.
```

Comment:
@RobertLaurella Can you double-check this sentence? It reads weird to me

Reply:
I suggested a small change below. When the line reads "You may also want to subclass it", it may be worth specifying what "it" refers to (the BaseModelCheckpoint class?)